addons/l10n_nl/views/partner_address_view.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="view_res_partner_dutch" model="ir.ui.view">
+        <field name="name">res.partner.form.address.nl</field>
+        <field name="model">res.partner</field>
+        <field name="arch" type="xml">
+            <form>
+                <div class="o_address_format">
+                    <field name="parent_id" invisible="1"/>
+                    <field name="type" invisible="1"/>
+
+                    <!-- Only shown in edit mode (after clicking on edit) -->
+                    <div class="o_address_format oe_edit_only">
+                        <div attrs="{'invisible': ['|', ('parent_id', '=', False), ('type', '!=', 'contact')]}">
+                            <b>Company Address:</b>
+                        </div>
+                        <div class="o_row">
+                            <field name="street_name" placeholder="Streetname..."
+                                   attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
+                        </div>
+                        <div class="o_row">
+                            <label for="street_number">House</label>
+                            <field name="street_number" colspan="1" col="1"
+                                   class="o_address_street"/>
+                            <label for="street_number2">Door</label>
+                            <field name="street_number2" colspan="1" col="1"
+                                   class="o_address_street"/>
+                        </div>
+                        <div class="o_row">
+                            <field name="street2" placeholder="Street2..." class="o_address_street"
+                                   attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
+                        </div>
+                        <div class="o_row">
```

Comment:
I am afraid there is no way with an xpath and view in primary mode instead of copying the entire view?  I saw for mx it also takes everything.

Reply:
I've searched around in Odoo and tried a few attempts but couldn't get to the desired result without copying the view content sadly. As far as I'm aware we can't.

Reply:
Why the difference between oe_edit_only and oe_readonly?  And really no possibilty to simply replace the zip with nothing and place it before or after something with an xpath?

Reply:
@jco-odoo I really couldn't find a way to manipulate the views to get what would be perfect for the Dutch in both readonly and edit mode. I've spent two hours trying without any success so then I decided to go with that. You're free to give it a try, I'd be really happy if you'd find a cleaner way. Otherwise I think we should just go with this ☺️